### ISSUE TITLE
[Question] Integrate VegaFusion

### ISSUE THREAD

**Issue by sweco-semtne** [livebook-dev/vega_lite#33]
Would it be feasible to implement something similar or integrate with VegaFusion.
It is a serverside accelerator for large datasets visualized with Vega

https://github.com/vegafusion/vegafusion/

**Comment by josevalim** [comment]
Yes, although I don't think we need a separate library or to use VegaFusion. Instead, using the Table.Reader protocol, we can ask the underlying table if it provides efficient aggregations. If it does, we can ask the underlying data structure to aggregate for us. This way we can automatically aggregate for you when using stuff like Explorer or other data structures.

**Comment by josevalim** [comment]
After more discussions on this, we decided to make this part of Kino.VegaLite. The smart cell will include in its output the filtered data if it detects and Explorer DataFrame.
